ddp-workspace/projects/ddp-pancan/src/app/components/auth/auth.component.scss

```diff
@@ -2,10 +2,22 @@
 
 .join-button {
     @extend .btn-auth;
-    margin-left: 15px;
+    margin-left: 0.9rem;
```

Comment:
I'm not sure we should convert to rem absolutely all pixels. Space between buttons can be considered as a constant otherwise it might be unpredictably small/big space

Reply:
It was the deliberate change. Also It have been checked for every screen breakpoint.